src/test/kotlin/no/nav/familie/dokument/pdf/PdfGenerartorTest.kt

```diff
@@ -0,0 +1,112 @@
+            StandardCharsets.UTF_8
+        )
+    }
+
+    /**
+     * Test if the data in the given byte array represents a PDF file.
+     */
+    fun isPdf(data: ByteArray): Boolean {
+        if (data.size > 4 && data[0] == 0x25.toByte() && // %
+            data[1] == 0x50.toByte() && // P
+            data[2] == 0x44.toByte() && // D
+            data[3] == 0x46.toByte() && // F
+            data[4] == 0x2D.toByte()
+        ) { // -
+
+            // version 1.3 file terminator
+            if (data[5] == 0x31.toByte() && data[6] == 0x2E.toByte() &&
+                data[7] == 0x33.toByte() && data[data.size - 7] == 0x25.toByte() && // %
+                data[data.size - 6] == 0x25.toByte() && // %
+                data[data.size - 5] == 0x45.toByte() && // E
+                data[data.size - 4] == 0x4F.toByte() && // O
+                data[data.size - 3] == 0x46.toByte() && // F
+                data[data.size - 2] == 0x20.toByte() && // SPACE
+                data[data.size - 1] == 0x0A.toByte()
+            ) { // EOL
+                return true
+            }
+
+            // version 1.3 file terminator
+            if (data[5] == 0x31.toByte() && data[6] == 0x2E.toByte() &&
+                data[7] == 0x34.toByte() && data[data.size - 6] == 0x25.toByte() && // %
+                data[data.size - 5] == 0x25.toByte() && // %
+                data[data.size - 4] == 0x45.toByte() && // E
+                data[data.size - 3] == 0x4F.toByte() && // O
+                data[data.size - 2] == 0x46.toByte() && // F
+                data[data.size - 1] == 0x0A.toByte()
+            ) { // EOL
+                return true
+            }
+        }
+        return false
+    }
+
+    private fun pdfsAreEqual(resource: String, actualPdfBytes: ByteArray): Boolean {
+        Files.createDirectories(Paths.get(TEST_OUTPUT_PATH))
+
+        // Load expected PDF document from resources, change class below.
+        val expectedPdfBytes = toByteArray("$PDF_RESOURSE_PATH/$resource")
+
+        // Get a list of results.
+        val problems = PdfVisualTester.comparePdfDocuments(
+            expectedPdfBytes, actualPdfBytes, resource, false
+        )
+
+        // Get a list of results.
+        if (problems.isNotEmpty()) {
+            System.err.println("Found problems with test case ($resource):")
+            System.err.println(problems.stream().map { p: PdfVisualTester.PdfCompareResult -> p.logMessage }
+                                   .collect(Collectors.joining("\n    ", "[\n    ", "\n]")))
+            System.err.println("For test case ($resource) writing failure artifacts to '$TEST_OUTPUT_PATH'")
+            File(TEST_OUTPUT_PATH, "$resource---actual.pdf").writeBytes(actualPdfBytes)
+        }
+        for (result in problems) {
+            if (result.testImages != null) {
+                var output = File(TEST_OUTPUT_PATH, resource + "---" + result.pageNumber + "---diff.png")
```

Comment:
Her burde vi bruke string-templates. For eksempel:
```suggestion
                var output = File(TEST_OUTPUT_PATH, "$resource---${result.pageNumber}---diff.png")
```